cli/displayAdsRecorder.js

```diff
@@ -12,28 +12,28 @@ const findAdsInDirectory = require("../src/util/findAdsInDirectory");
 (async () => {
   program
     .version(packageJson.version)
-    // .option('-g, --glob <data>', 'Globbing pattern like "-p ./src/**/.richmediarc"')
     .option('-c, --chunkSize <data>', 'Define chunkSize', 10)
     .option('-t, --targetDir <data>', 'Set target dir')
-    .option('-b, --backup <data>', 'If you want to run backup images only', 0)
     .option('-a, --all', 'If you want to record all', false)
+    .option('-j, --jpg [size]', 'If you want to output jpg and optional kbs')
+    .addOption(
+      new program.Option("-g, --gif [loop]", "If you want to output animated gifs and loop them or play once").choices(['-1', '0'])
```

Comment:
It turns out that when you do --gif -1, it will throw an error saying -1 is not a variable. We can do --gif=-1, but it is a bit error prone. Maybe we should rename (loop/once) it and have a default? I also noticed that when you only add gif, it still renders out video, that makes sense in a way that the video is needed for the gif. But maybe we should rename the progressbar with 'rendering video for gif' or something?

Reply:
I agree that we must avoid -1 and go with loop/once. But default - no. Let's stick to what we have everywhere